docs/src/design/decoder/constraints.md

```diff
@@ -128,10 +130,10 @@ $$
 In the above, $a$ represents the address value in the decoder which corresponds to the hasher chiplet address at which the hasher was initialized (or the last absorption took place).  As such, $a + 7$ corresponds to the hasher chiplet address at which the result is returned.
 
 $$
-f_{ctrli} = f_{join} + f_{split} + f_{loop} +  f_{call} + f_{syscall} \text{ | degree} = 5
+f_{ctrli} = f_{join} + f_{split} + f_{loop} +  f_{call} \text{ | degree} = 5
 $$
 
-In the above, $f_{ctrli}$ is set to $1$ when a control flow operation that signifies the initialization of a control block is being executed on the VM.  Otherwise, it is set to $0$.
+In the above, $f_{ctrli}$ is set to $1$ when a control flow operation that signifies the initialization of a control block is being executed on the VM.  Otherwise, it is set to $0$. An exception is made for the `SYSCALL` operation. Although it also signifies the initialization of a control block, it must additionally send a procedure access request to the [kernel ROM chiplet](../chiplets/kernel_rom.md) via the chiplets bus. Therefore, it is excluded from this flag and its communication with the chiplets bus is handled separately.
```

Comment:
Debatable, I think. I left it there because it's true that it initializes a control block and everything described in that paragraph needs to be done for it. If I remove it from that paragraph, then we need to describe it again for `SYSCALL`, which we could do. I tried to address it with the paragraph below instead, but if you want me to remove it from 146 I can

Reply:
No - let's keep it then.